src/uct/api/uct.h

```diff
@@ -1012,6 +1037,12 @@ struct uct_iface_params {
     void                                         *rndv_arg;
     /** Callback for tag matching unexpected rndv messages */
     uct_tag_unexp_rndv_cb_t                      rndv_cb;
+
+    void                                         *async_event_arg;
+    /** Callback for asynchronous event handling. The callback will be
+     * invoked from UCT transport when there are new events to be
+     * read by user */
```

Comment:
.. if the iface has `@ref` XXYY capability

Reply:
done